plugins/org.locationtech.udig.context/src/org/locationtech/udig/context/internal/ContextExportWizard.java

```diff
@@ -91,9 +91,9 @@ static void writeContext(IMap map, BufferedWriter out) throws IOException {
         append(2, out, "<ResourceList>"); //$NON-NLS-1$
         for (ILayer layer : map.getMapLayers()) {
             try {
-                if (layer.isType(Layer.class)) {
+                if (layer.hasResource(Layer.class)) {
                     writeLayer(layer, out);

```

Comment:
wondering what would happen here, since WMSLayer.hasResource() is always false whereas isType was true ..

Reply:
Basically WMSLayer.java is located into the module "org.locationtech.udig.info.tests" - so it's a test class. Does it really that matter? From my perspective we may even think about removing this class (should not belong to a test bundle).